benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -428,6 +425,7 @@ async def benchmark(
         )
       )
     tasks.append(task)
+    prompts_sent += 1
```

Comment:
Is the async for loop thread safe? Do we need a lock on this prompts_sent?

Reply:
Read into the library a bit more, it looks like this should work as is - eg async for loop is iterative and will wait for the next generated request before moving on